documentation/Learn-Beton/Frequently-Asked-Questions.md

```diff
@@ -161,6 +161,20 @@ These events will set a specific tag. Now when the player starts the conversatio
 should check (using the `globaltag` condition) which tag is currently set and give the player different quests based on that.
 Of course, the static folder event also needs to remove the the current tag before setting a new one.
 
+## Global Quests (all players work together)
+
+There is no super easy way to do this yet. One would do this:
+Create an objective that is immediately fired upon the first action (this means setting the amount to one for most objectives).
+That objective must be `persistent` so it restarts immediately upon completion.
+It would trigger a `globalpoint` event that increases some globalpoint variable by one.
+That globalpoint variable tracks the players combined progress as it is the same for all of them.
+Therefore, there must also be some sort of `globalpoint`condition that checks whether the "global objective" has been completed.
+It must be checked in a "checkCompletion" event that is run alongside the globalpoint event. It could look like this:
+
+```YAML
+checkCompletion: "run ^objective delete globalQuest ^folder giveRewards,notifyRewards conditions:globalObjectiveDone"
```

Comment:
`objective delete globalQuest` will remove the objective only from the completing player!

A better global-quest flow might be:
objective:
```yaml
# collect cobble stone to progress the global quest
gquest: block COBBLESTONE -1 events:gquest_progress conditions:gquest_active,!gquest_complete global persistent
```
event:
```yaml
# first increment, then check the completion, the completion is delayed by a tick
gquest_progress: folder gquest_increment_counter,gquest_check_completion period:1ticks
# increment the globalpoint counter
gquest_increment_counter: globalpoint gquest 1
# run completion events on completion
gquest_check_completion: folder gquest_notify,gquest_on_completion condition:gquest_complete
```
conditions:
```yaml
# somehow determine the start of the global quest, e.g. with a globaltag or a `partialdate` condition
gquest_active: globaltag gquest_active
# complete at one million collected
gquest_complete: globalpoint gquest 1000000
```